app/mailers/collections_mailer.rb

```diff
@@ -85,6 +87,7 @@ def version_draft_created
     @user = UserPresenter.new(user: params[:user])
     @collection_version = params[:collection_version]
     @owner = params[:owner]
+    @work = params[:work_version].work
 
```

Comment:
we need the work available in the views, the work is available via the work_version from the observer, see https://github.com/sul-dlss/happy-heron/blob/main/app/services/collection_observer.rb#L5-L27